vue/components/personalization.vue

```diff
@@ -207,7 +204,7 @@ export const personalization = {
             this.initialOptions = Object.assign({}, options);
         });
 
-        this.$bus.bind("initials_extra", initialsExtra => {
+        this.$bus.bind("initials_change", initialsExtra => {

```

Comment:
why the name change? If not properly justified this is a hot spot for regressions. Please elaborate on the semantics of each of the events.

Reply:
Yup, I've just analyzed this, there's a huge difference in semantics. This is not a proper fix.

Reply:
Initials changed = the UI initials changed
Initials extra = the SDK initials changed

In here we want to subscribe to changes in the UI definition, to keep the several instances in sync.